about.html

```diff
@@ -433,13 +455,13 @@ <h4 class="title">
                             Information
                         </div>
                         <small class="text mb-10 d-block">
-                            x Thatcher Road St, Brookly, <br> x,sydney,nsw,Australia
+                            348 Waterloo Road, GREENACRE, NSW, 2190 <br> 
                         </small>
                         <small class="text mb-10 d-block">
-                            +0011111222
+                            (+61)0417 90 6971
```

Comment:
Can you please format this as follows?

![image](https://user-images.githubusercontent.com/8443215/229262194-127ccd5c-c9f4-43d7-a48c-b41821d2889c.png)

Source: https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/numbers-and-measurements/telephone-numbers#use_chunks_of_digits_to_help_people_read_telephone_numbers